monai/handlers/classification_saver.py

```diff
@@ -77,6 +85,12 @@ def __call__(self, engine: Engine) -> None:
         Args:
             engine: Ignite Engine, it can be a trainer, validator or evaluator.
         """
-        meta_data = self.batch_transform(engine.state.batch)
-        engine_output = self.output_transform(engine.state.output)
-        self.saver.save_batch(engine_output, meta_data)
+        _meta_data = self.batch_transform(engine.state.batch)
+        if "filename_or_obj" in _meta_data:
```

Comment:
The string "filename_or_obj" is used in a number of places here and in other code. This should be stored somewhere as a constant value with documentation stating what it's used to store.

Reply:
Thanks for your suggestion, let me change to some `constant` value.

Reply:
updated in the latest commit.
Thanks.